### DISCUSSION TITLE
missing INDI.NAME.LANG ?

### DISCUSSION THREAD

**Discussion by albertemmerich** [FamilySearch/GEDCOM#108] [Q&A] [answered]
starting to use the new GEDCOM 7 version of program, I am asking why INDI.NAME does not have a substructure LANG at all, but for every substructure TRAN you must give LANG. By this we cannot document in which language the (main) NAME is given.
I have a problem with this, as some of my ancestors lived in French occupied area, and the documents show French names. If I take the original name from document to be the first, I cannot add it to be French.

This gives some more problems, as checking the sex of a person against its givn names depends on the language. French "Auguste" is male, German "Auguste" is female. So if I cannot put the language to the NAME of a person in a structured way, any automatic check for conforming sex to givn will not work.

**Comment by dthaler** [comment, marked as answer]
I believe this is now tracked in issue #167 as a feature request we will consider for the next minor release (7.1).
